docs/DeveloperGuide.md

```diff
@@ -4,34 +4,122 @@
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
+Main structure of the code and the parse feature is adapted from [here](https://github.com/se-edu/addressbook-level2).
+
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Main structure
+
+
+### Architecture
+{insert diagram to show architecture of code}
```

Comment:
Im not sure if I just missed it or its a work in progress but I didn't see the diagram in the initial DG